File: yh_turtle/src/yh_turtle_keyboard.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

ros::Publisher pub;

void keyboardCallback(const geometry_msgs::Twist::ConstPtr& msg)
{
    pub.publish(msg);
}

int main(int argc, char** argv)
{
    ros::init(argc, argv, "yh_tutle_keyboard");
    ros::NodeHandle nh;
    ros::Subscriber sub = nh.subscribe("/cmd_vel", 1, keyboardCallback);
    pub = nh.advertise<geometry_msgs::Twist>("/turtle1/cmd_vel", 1);

    ros::spin();
    return 0;

}

File: yh_turtle/scripts/yh_turtle_keyboard.py
#!/home/min/anaconda3/bin/python
# -*- coding: utf-8 -*-

import rospy
from geometry_msgs.msg import Twist

pub = rospy.Publisher("/turtle1/cmd_vel", Twist, queue_size=1)

def keyboardCallBack(msg):
    pub.publish(msg)

def listner():
    rospy.init_node("yh_turtle_keyboard", anonymous=True)
    rospy.Subscriber("/cmd_vel", Twist, keyboardCallBack)

    rospy.spin()

if __name__ == "__main__":
    listner()